app/layouts/Public/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Lets incorporate this in the Dashboard Layout only. Pass public prop from createPage to Dashboard layout and handle it.

Reply:
ok fine